accessing
foreignError: anObject
	foreignError := anObject